mkdocs.yml

```diff
@@ -38,6 +38,7 @@ plugins:
             show_submodules: true
   - table-reader
 markdown_extensions:
+  - callouts
```

Comment:
This is a little helper that makes "admonitions" look just OK in both GitHub and mkdocs. It's not perfect, but it is better than not having it. So far, this only impacts `CONTRIBUTING.md` because that's the only file that has to look good rendered in both ways.